### ISSUE TITLE
Error when # nearest items wanted exceeds items in rbush

### ISSUE THREAD

**Issue by hulbert** [mourner/rbush-knn#1]
I get this error when doing something like `var neighbors = knn(tree, [coords.lng, coords.lat], 50)`

```
/repo/node_modules/rbush-knn/index.js:23
        while (queue.peek()._knnItem && result.length < n) result.push(queue.p
                           ^
TypeError: Cannot read property '_knnItem' of undefined
    at knn (/repo/node_modules/rbush-knn/index.js:23:28)
```

**Comment by mourner** [comment]
Right, thanks for the report. Pretty easy to fix.

**Comment by mourner** [comment]
Fixed and published 1.0.1.

**Comment by hulbert** [comment]
:+1: thanks!
